src/audio/kpb.c

```diff
@@ -424,6 +426,13 @@ static int kpb_prepare(struct comp_dev *dev)
 		}
 	}
 
+	if (!kpb->sel_sink || !kpb->host_sink) {
+		trace_kpb("kpb_prepare() error: could not find "
+			  "sinks: sel_sink %d host_sink %d",
+			  (uint32_t)kpb->sel_sink, (uint32_t)kpb->host_sink);
```

Comment:
This is potential exception, when one of them will be NULL.

Reply:
@tlauda what do you mean? If any sink is null it will just be casted to 0

Reply:
Of course. I don't know what I was thinking...